app/modules/notifications/models.py

```diff
@@ -16,10 +15,18 @@ class NotificationStatus(str, enum.Enum):
     unread = 'unread'
 
 
-class NotificationMessageTemplate(str, enum.Enum):
+class NotificationType(str, enum.Enum):
     raw = 'raw'
-    new_enc = 'new_enc'
-    collab_request = 'collab_request'
+    new_enc = 'new_enc'  # TODO what's this for?
```

Comment:
@jon, what was this for again?

Reply:
this is to let a user know when there is a **new sighting** of an animal "they are following".  now, what does that part in quotes mean exactly?  good question!  i think in old world what it meant is that you were the person who submitted an encounter that was identified as individual X.    will elaborate in slack....

Reply:
i guess we can drop this now that its not mvp